.github/workflows/apply-pr665-fixes.yml

```
@@ -0,0 +1,451 @@
+          replace_once(
+              "app/cache.py",
+              '''    def record_success(self) -> None:
+                  """Record successful request."""
+                  if self._state == CircuitState.HALF_OPEN:
+                      logger.info("Circuit '%s': HALF_OPEN -> CLOSED", self.name)
+                      self._state = CircuitState.CLOSED
+                      self._failure_count = 0
+                  elif self._state == CircuitState.CLOSED:
+                      self._failure_count = 0
+
+''',
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)